cmd/harmony/flags_test.go

```diff
@@ -499,6 +541,138 @@ func TestP2PFlags(t *testing.T) {
 				ConnManagerLowWatermark:  defaultConfig.P2P.ConnManagerLowWatermark,
 				ConnManagerHighWatermark: 400,
 				WaitForEachPeerToConnect: false,
+				NoTransportSecurity:      defaultConfig.P2P.NoTransportSecurity,
+				NAT:                      defaultConfig.P2P.NAT,
+				UserAgent:                defaultConfig.P2P.UserAgent,
+				DialTimeout:              defaultConfig.P2P.DialTimeout,
+				Muxer:                    defaultConfig.P2P.Muxer,
+				NoRelay:                  defaultConfig.P2P.NoRelay,
+			},
+		},
+		{
+			args: []string{"--p2p.connmgr-high", "400"},
```

Comment:
could it be called in another way? it has terrible naming

Reply:
maybe we could but for now if this proposed update is not critical, let's keep it for the next release.